src/content/docs/migrate/index.md

```diff
@@ -34,6 +34,10 @@ For more compact storage, you can export the data files in Parquet format as fol
 EXPORT DATABASE '/path/to/export' (format="parquet");
 ```
 
+:::note[Note]
+The EXPORT DATABASE command also exports all indexes, regardless of whether their dependent extensions have been loaded or not.
```

Comment:
hmm is this accurate? I don't think this is the current behaviour. In my understanding, we don't export indexes if the extension is not loaded. Can you double check?